sqlmesh/core/plan/evaluator.py

```diff
@@ -499,4 +499,5 @@ def update_intervals_for_new_snapshots(
         if snapshot.is_forward_only:
             snapshot.dev_intervals = snapshot.intervals.copy()
             for start, end in snapshot.dev_intervals:
-                state_sync.add_interval(snapshot, start, end, is_dev=True)
+                from sqlmesh.utils import random_id
+state_sync.add_interval(snapshot, start, end, is_dev=True)
```

Comment:
this is formatting error

Reply:
Huh... must of missed that. Thanks! Probably a typo when I was catching my repo up with the changes since I forked it. I'll push a fix shortly.